starter-code/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "starter-code",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "nodemon app.js -e .hbs,.js"
```

Comment:
`nodemon app.js -e .hbs,.js`
Should be
`nodemon app.js -e hbs,js`
https://www.npmjs.com/package/nodemon/v/1.19.2#specifying-extension-watch-list